tasks.py

```diff
@@ -1340,7 +1340,7 @@ def e2etest(
     )
 
     if export != None:
-        run("{} export logs {}".format(kind_path, export))
+        run("{} export logs {}".format(kind_path, export), warn=True)
```

Comment:
It possible also to print the command? I think there is echo true something?

Reply:
yep but should be part of the warning I guess